source/contributor-ladder.md

```diff
@@ -0,0 +1,122 @@
+---
+title: ASF Contributor Ladder
+---
+
+Projects at the ASF offer a way to grow in your responsibilites in the
+project, from being a user, all the way up to being a maintainer of the
+project. While the specific details may look different from one project
+to another, the rungs of the ladder look mostly the same.
+
+* [User](#user)
+* [Contributor](#contributor)
+* [Committer](#committer)
+* [PMC Member](#pmc-member)
+* [ASF Member](#asf-member)
+* [Officers of the Foundation](#officers-of-the-foundation)
+* [Board of Directors](#board-of-directors)
+
+## User
+
+Everything that we do in an ASF project is for the user. As a user of an
+Apache project, your responsibilities are just to enjoy the software.
+
+We encourage you to join the user mailing list to ask questions, and
+make suggestions. Members of the project are there to help out when you
+run into problems. You can find this list (or lists) by looking under the
+project name on [lists.apache.org](https://lists.apache.org).
+
+As a user, you're expected to abide by the project's [code of
+conduct](https://www.apache.org/foundation/policies/conduct), so that
+interactions are polite, friendly, and helpful.
+
+As you become a power user of the project, you may consider becoming a
+contributor.
+
+## Contributor
+
+While most people think of open source contributors as being coders,
+there are a wider variety of ways that you can contribute to a project.
+
+Most users' first contribution will be to open a ticket on the project's
+issue tracker, or to answer a less experienced user's questions. These
+actions help shape the future direction of the project, and are
+important contributions.
+
+We encourage you to join the project's developer mailing list at this
+time, to engage with the dicussions about the direction that the project
+is taking. You can find this list (or lists) by looking under the
+project name on [lists.apache.org](https://lists.apache.org).
+
+Other contributions may include design, promotion, documentation
+improvements, events, and may other things.
```

Comment:
Line 51: might be better as 'improvements, organizing events, and many other things.' Corrects typo, too